ethers-core/src/types/withdrawal.rs

```diff
@@ -0,0 +1,29 @@
+use crate::types::{Address, U256, U64};
+use serde::{Deserialize, Serialize};
+
+/// A validator withdrawal from the consensus layer.
+/// See EIP-4895: Beacon chain push withdrawals as operations.
+#[derive(Debug, Clone, Default, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Withdrawal {
+    /// Monotonically increasing identifier issued by consensus layer
+    pub index: U64,
+
+    /// Index of validator associated with withdrawal
+    pub validator: U64,
```

Comment:
add serde name to `validatorIndex`:

```json
    {
      "index": "0x1899c",
      "validatorIndex": "0x52b11",
      "address": "0xb9d7934878b5fb9610b3fe8a5e441e8fad7e293f",
      "amount": "0x51ae3c69"
    }
```

Reply:
or probably better to rename the field to `validator_index` as well

Reply:
ok, renaming it rn